src/status_im/ui/screens/screens.cljs

```diff
@@ -1,7 +1,7 @@
 (ns status-im.ui.screens.screens
   (:require
     [quo.design-system.colors :as colors]
-    [utils.i18n :as i18n]
+    [quo2.foundations.colors :as quo2.colors]
```

Comment:
I had to add quo2.colors because it;s required the NavigationBar in android to be black.